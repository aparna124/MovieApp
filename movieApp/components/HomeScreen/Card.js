import React from 'react';
import { View, Text, Image, Dimensions} from 'react-native';
const IMAGE_API="https://image.tmdb.org/t/p/w185/";

export default function Card({ titel, poster_path, vote_average, release_date, overviewposter_path }) {

    const ScreenWidth = Dimensions.get("window").width;

    return (
        <View style={{justifyContent: 'center'}}>
                <Image
                    style={{
                        width: (ScreenWidth - 40) / 2,
                        height: 200,
                        // margin: 10,
                        borderRadius: 5,
                        shadowColor: "#051934",
                        marginVertical: 5,
                    }}
                    source={{ uri: IMAGE_API + poster_path }}
                />
        </View>
    );
}