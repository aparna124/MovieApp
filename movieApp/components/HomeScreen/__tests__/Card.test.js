import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../Card';

describe('<Card />', () => {
  test('card renders correctly', () => {
    const tree = renderer.create(<Card />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});