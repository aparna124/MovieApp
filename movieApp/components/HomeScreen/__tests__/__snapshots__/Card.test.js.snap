// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> card renders correctly 1`] = `
<View
  style={
    Object {
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://image.tmdb.org/t/p/w185/undefined",
      }
    }
    style={
      Object {
        "borderRadius": 5,
        "height": 250,
        "marginVertical": 5,
        "shadowColor": "#051934",
        "width": 355,
      }
    }
  />
</View>
`;
